### PR TITLE
modified Assignment 1,2,3,4

### PR DESCRIPTION
**Pull request by Jestinxavier**

### CODE REVIEW

**Review by mubeen2710 — APPROVED** [review]

**Review by Richa199 — APPROVED** [review]

File: Jestin/HTML/Assignment4/ass4q2.html [hunk 1]
```diff
@@ -1,110 +1,93 @@
...
+                        <td class="w"></td>
+                    </tr>
+                    <tr class="adc" >
+                        <th>4:30</th>
+                        <td>&nbsp;</td>
+                        <td>&nbsp;</td>
+                        <td>&nbsp;</td>
+                    </tr>    
+            </table>
+    </body>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q2.html:126]: add new line at the end of file

File: Jestin/HTML/Assignment4/ass4q3.html [hunk 2]
```diff
@@ -1,90 +1,83 @@
...
+                    <td class="c">5:00 p.m.</td>   
+                </tr>
+                <tr>
+                    <td>Wednesday</td>
+                    <td class="b">8:00 a.m.</td>
+                    <td class="d">12:00 p.m.</td>
+                    <td>XSL Formatting Object</td>
+                </tr>
+        </table>
+    </body>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q3.html:82]: add new line at the end of file

File: Jestin/HTML/Assignment3/new (copy).html [hunk 3]
```diff
@@ -1,41 +1,36 @@
...
+            </div>
+        </div>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis, nam sapiente dignissimos sint voluptates assumenda aliquid fugit. Perferendis eaque cumque distinctio libero, ad reiciendis similique qui voluptates nihil temporibus aut.</p>
+        <h3>Home Page subheading</h3>
+        <p> This is a home page sub heading. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Fugiat officia, rerum harum alias, libero magnam velit itaque, perspiciatis voluptatibus aperiam maiores exercitationem dicta? Modi suscipit perspiciatis consequatur hic nobis optio! This is a home page sub heading. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Repellendus, quaerat, asperiores quod fugit tempora voluptates iusto accusamus natus reprehenderit ea, quia facere maxime atque accusantium dolorem ab? Nihil, placeat alias? Lorem ipsum dolor sit amet consectetur, adipisicing elit. Blanditiis assumenda saepe tenetur expedita earum magnam inventore accusantium itaque sunt est architecto accusamus omnis aspernatur fuga porro labore, deleniti asperiores aliquam.</p>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment3/new (copy).html]: add new line at the end of file

File: Jestin/HTML/Assignment4/ass4q1.html [hunk 4]
```diff
@@ -1,82 +1,62 @@
...
+            </tr>
+            <tr>
+                <th>Female</th>
+                <td>1.9</td>
+                <td>0.003</td>
+                <td>43%</td>
+            </tr>
+       </table>         
+    </body>
 </html>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q1.html]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Jestin/HTML/Assignment4/ass4q1.html (hunk 4 above)
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q1.html]: add new line

File: Jestin/HTML/Assignment4/ass4q2.html [hunk 5]
```diff
@@ -1,110 +1,93 @@
...
 </html>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q2.html]: add new line at the end of file

File: Jestin/HTML/Assignment4/ass4q3.html [hunk 6]
```diff
@@ -1,90 +1,83 @@
...
 </html>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q3.html]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Jestin/HTML/Assignment3/assig q1.html [hunk 7]
```diff
@@ -1,72 +1,60 @@
...
-   </style>
-   <ol>
-       <li style="color: blue;">MI-MEAN
-        <ol class="roman"><li> MI-MEAN-6  
-            <ul class="a">
-                <li>Shweta</li>
-                <li>Ajay</li>
-                <li>Trupti</li>
+<html>
+    <body>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment3/assig q1.html:6]: head tag is missing

File: Jestin/HTML/Assignment3/mi11.html [hunk 8]
```diff
@@ -1,32 +1,33 @@
...
-    </div>
-    
-    
-</body>
\ No newline at end of file
+            </div>
+            <p>   Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolores expedita quam, excepturi voluptates nobis voluptatum enim a consectetur voluptatem. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tenetur amet explicabo facere pariatur beatae quod eligendi aut? Ipsum, nam tempore natus ea sequi pariatur iste in, ipsam enim molestiae aut? Pariatur unde sequi nisi corporis voluptates suscipit inventore alias iste harum. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Recusandae eaque,  <br> 
+        </div>
+        
+
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment3/mi11.html]: remove extra lines

File: Jestin/HTML/Assignment3/new (copy).html [hunk 9]
```diff
@@ -1,41 +1,36 @@
...
-        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Blanditiis assumenda saepe tenetur expedita earum magnam inventore accusantium itaque sunt est architecto accusamus omnis aspernatur fuga porro labore, deleniti asperiores aliquam.
-    </p>
-</body>
\ No newline at end of file
+<html>
+    <head>
+        <title>Assignment 3 q1</title>
+        <style>
+            h1{text-align: center;}
+
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment3/new (copy).html]: remove extra lines

File: Jestin/HTML/Assignment4/ass4q2.html [hunk 10]
```diff
@@ -1,110 +1,93 @@
...
+            .adc{
+                border-bottom: 10px solid gray;
+            }
+            .w{
+                border-top: transparent;
+            }
+    </style>
+    </head>
+    <body>
+            <table style="width: 100%;" border="20" cellpadding="23">
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment4/ass4q2.html]: to much space

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**